core/src/main/scala/filodb.core/memstore/OnDemandPagingShard.scala

```diff
@@ -72,25 +76,30 @@ TimeSeriesShard(dataset, storeConfig, shardNum, rawStore, metastore, evictionPol
       }
     }
     shardStats.partitionsQueried.increment(inMemoryPartitions.length)
-
+    val span = Kamon.buildSpan(s"odp-shard-rawchunks-loadtime")
+      .withTag("dataset", dataset.name)
+      .withTag("shard", shardNum)
+      .start()
+    inMemoryPartitionsMetric.increment(inMemoryPartitions.length)
+    opdPartitionsMetric.increment(methods.length)
     logger.debug(s"dataset=${dataset.ref} shard=$shardNum Querying ${inMemoryPartitions.length} in memory " +
       s"partitions, ODPing ${methods.length}")
 
     // NOTE: multiPartitionODP mode does not work with AllChunkScan and unit tests; namely missing partitions will not
     // return data that is in memory.  TODO: fix
-    if (storeConfig.multiPartitionODP) {
+    (if (storeConfig.multiPartitionODP) {
       Observable.fromIterable(inMemoryPartitions) ++
       Observable.fromTask(odpPartTask(indexIt, partKeyBytesToPage, methods, chunkMethod)).flatMap { odpParts =>
         val multiPart = MultiPartitionScan(partKeyBytesToPage, shardNum)
         shardStats.partitionsQueried.increment(partKeyBytesToPage.length)
```

Comment:
We also need to revive and increment the chunks-paged-in metric. This was once being updated, but is no longer updated now.